CHANGELOG.md

```diff
@@ -7,18 +7,15 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## Unreleased
 
+## [0.38.1] - 2023-06-02
+
 ### Added
 
 - Remote test to verify that, with the right permission, we can test on an org from the training model of an other organization (#237)
 
 ### Changed
 
 - chore: rename tuple to task (#239)
-
-## [0.38.0] - 2023-01-30
-
-### Changed
-
```

Comment:
Are we supposed to remove this? I'm not sure as to the status of 0.38.0 but there is a GH release to that name so we should keep it in the changelog.

Reply:
Yes you're right thanks ! :)